src/redirect.js

```diff
@@ -0,0 +1,33 @@
+/**
+* If you use a Smartcar hosted redirect (by selecting the option in the
+* developer dashboard) this code is served automatically. If you host your own
+* redirect you can source this file (along with using the Javascript SDK to
+* close out the redirect and trigger your specified `onComplete` function.
+*/
+
+(function() {
+  'use strict';
+  if (!window.opener) {
+    throw new Error('window.opener must be defined');
+  }
+
+  const params = new URLSearchParams(window.location.search);
+
+  const maybeOriginParam = params.get('origin');
```

Comment:
using 'origin' for parameter name of target domain to post to. anyone have other/preferred names for this?

Reply:
Anyone else have thoughts on `origin` param name?